docker/server/aws_imdsv2.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# Metadata service URL variable
```

Comment:
can you add a paragraph at the top of this script that explains what it is doing and what the user would need to set?